.github/ISSUE_TEMPLATE/feature_request.md

```
@@ -2,10 +1,17 @@
+---
+name: 🚀 기능 추가
+about: 새로운 기능 개발/구현을 위한 이슈 또는 PR에 사용합니다.
+title: '[feat] 구현기능 간략설명'
+labels: feature
+assignees: ''
+---
+## 어떤 기능을 추가하셨나요?
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)